products/libllbuild/include/llbuild/buildsystem.h

```diff
@@ -24,18 +24,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#ifdef _MSC_VER
-typedef int llbuild_pid_t;
-#else
-
-#if defined(__linux__) || defined(__GNU__)
-#include <termios.h>
-#else
-#include <sys/types.h>
-#endif
-
-typedef pid_t llbuild_pid_t;
-#endif
+#include "llbuild/Basic/CrossPlatformCompatibility.h"
```

Comment:
These are here because the C library headers are distinct from the C++ headers and thus should not depend on that header being available.